README.md

```diff
@@ -1,2 +1,28 @@
 # cli-github-users
 LovelyStay challenge
+
+Hi lovely mates hope you're well :)
+
+Let me describe a little bit what I did and how to use it.
+
+First of all, set your project using npm start.
+Secondly, set your .env file using the .envexample (using in the same path only) or use setenv command (described below).
+
+The main function is called gitHubUsers and as soon you call it with --help command it shows a list of commands.
+For each command it's possible to call --help command to see what arguments are available.
+
+Functions:
+
```

Comment:
Assuming that the env approach was supposed to work (but it doesn't, it complains when I run `npm run test`), these commands give no feedback that they are not working

Reply:
I changed some steps on Readme, could you try again?